packer/files/pull_traefik_image.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Check if docker command is available
+if ! command -v docker &> /dev/null; then
+    echo "ERROR: docker command not found. Please install Docker first." >&2
+    exit 1
+fi
+
+TAG="${TAG:-v3.5.6}"
```

Comment:
Do we have any plan to upgrade the version?
Is this something we should do manually before each monitor release? Or maybe we should fetch here the latest stable version?